layouts/partials/footer.html

```diff
@@ -1,21 +1,43 @@
 {{ $copy := site.Copyright }}
 {{ $year := now.Year }}
 {{ $logo := printf "img/logos/%s" site.Params.logos.footer | relURL }}
-<footer class="footer has-background-black">
+<footer class="footer has-background-white">
```

Comment:
White is the default, so you don't need to specify `has-background-white`

Reply:
When removing the white background I found that the footer has some sort of grey as default, I like that, so we'll keep the grey.